lib/utils/collect-keys.ts

```diff
@@ -210,13 +210,11 @@ export function collectKeysFromAST(
         }
       } else {
         if (
-          (node.key.name === 'path' &&
-            (node.parent.parent.name === 'i18n' ||
-              node.parent.parent.name === 'i18n-t')) ||
-          (node.key.name === 'keypath' && node.parent.parent.name === 'i18n-t')
+          (node.key.name === 'path' && (['i18n', 'i18n-t', 'i18nt'].includes(node.parent.parent.name)) ||
+          (node.key.name === 'keypath' && ['i18n-t', 'i18nt'].includes(node.parent.parent.name)))
```

Comment:
Could you please use rawName to determine more accurately?

```suggestion
          (node.key.name === 'path' &&
            (node.parent.parent.name === 'i18n' ||
              node.parent.parent.name === 'i18n-t' ||
              node.parent.parent.rawName === 'I18nT')) ||
          (node.key.name === 'keypath' &&
            (node.parent.parent.name === 'i18n-t' ||
              node.parent.parent.rawName === 'I18nT'))
```

Reply:
Thanks for the suggestion, I've fixed it accordingly.